.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+For bug reports, please respect the following guidelines and check the boxes accordingly.
+
+For questions of the type: "How can this be done?", please consider posting them on Stackoverflow.
+
+For everything else ignore this template.
+
+## Bug report
+
+- [ ] Start a new R session
```

Comment:
```R
library("mlr")
parallelStartMulticore(2)
parallelMap(function(x) resample("classif.IBk", pid.task, cv5), 1:2, simplify=FALSE)
```
works if run in `R --vanilla`, but does not work when run in an R session that has an "RWeka" model in its save file. If you do the above after running
```R
library("mlr")
m = train("classif.IBk", pid.task)
q("yes")
```
it will hang.